docs/source/ui/pages/run_management.md

```diff
@@ -33,150 +40,153 @@ Step 1: Select runs on the runs table:
 
 <img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Runs Explorer Batch archive" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/runs-explorer/select_runs.png">
 
-Click on the `Archive` button. Fonfirmation popup appears. Click `Archive` again and the runs are archived!
+Click on the "Archive" button. A confirmation pop-up appears. Click "Archive" again and the runs are archived!
```

Comment:
I think code blocks is for showing the code snippet. Since, "Archive" is a button name, but not code block, it's correct to use double or single quotes (it depends on the format we use).